### DISCUSSION TITLE
what paper is Coqui based on?

### DISCUSSION THREAD

**Discussion by lmessinger** [coqui-ai/STT#1862] [General]
Hi

would be grateful for a link to a paper, or to an explanation of the core architecture (or both...)

many thanks!
Lior

**Comment by NanoNabla** [comment]
Hi,
as far as I know, there is no paper about coqui STT, yet. 
The core architecture should be the same as Mozillas DeepSpeech, [Baidus DeepSpeech network](https://arxiv.org/abs/1412.5567)

**Reply by jeremiahrose** [reply]
Yes, Coqui is based on that Baidu Deepspeech paper. Confirmed here in docs: https://stt.readthedocs.io/en/latest/playbook/ABOUT.html
